common/source/docs/common-nooploop.rst

```diff
@@ -63,6 +63,7 @@ Ensure that Copter-4.1 (or higher) is loaded onto the autopilot and connect with
 - set :ref:`EK3_SRC1_POSZ <EK3_SRC1_POSZ>` to 4 ("Beacon") to use NoopLoop for altitude or leave at "1" to use the barometer
 - set :ref:`EK3_SRC1_VELXY <EK3_SRC1_VELXY>` to 0 ("None")
 - set :ref:`EK3_SRC1_VELZ <EK3_SRC1_VELZ>` to 0 ("None")
+- set :ref:`EK3_SRC1_YAW<EK3_SRC1_YAW>` to 6 ("ExternalNAV")
```

Comment:
EK3_SRC1_YAW should be 1 (Compass) because sadly these beacon systems (pozyx, nooploop, marvelmind) can't provide yaw.

Thanks!